### PR TITLE
hydrus: 497 -> 500

### PR DESCRIPTION
**Pull request by D4ndellion**
###### Description of changes
Weekly update of hydrus

Also upgrades python-mpv, since this seems to be working fine

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by onny** [comment]
Result of `nixpkgs-review pr 190204` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>sublime-music</li>
  </ul>
</details>
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>hydrus</li>
    <li>jellyfin-mpv-shim</li>
    <li>plex-mpv-shim</li>
    <li>python310Packages.mpv</li>
    <li>python39Packages.mpv</li>
  </ul>
</details>

**Comment by onny** [comment]
One dependency, `sublime-music` fails after the mpv upgrade. Maybe you can fix this by updating or patching it?

**Comment by dali99** [comment]
@sumnerevans I dont have any airsonic setup to test whether or not this relaxation is fine. So if you could test that for me that would be great

**Comment by onny** [comment]
> @sumnerevans I dont have any airsonic setup to test whether or not this relaxation is fine. So if you could test that for me that would be great

Created an upstream issue https://gitlab.com/sublime-music/sublime-music/-/issues/343
Maybe it's save to relax the dependency and maybe we could integrate the upstream patch too :D

**Comment by sumnerevans** [comment]
I tested with changing the dependency to `^1.0.1` and it works fine. Let's patch it here and I'll work on trying to get it integrated upstream. There's some other dependency issues that I'm trying to figure out (Python packaging is :vomiting_face:)

**Comment by dali99** [comment]
Relaxing the dependency like currently done seems a lot better to me than pulling in a patch. But I'm open to be persuaded

**Comment by onny** [comment]
> Relaxing the dependency like currently done seems a lot better to me than pulling in a patch. But I'm open to be persuaded

Yeah that's fine. If you like you could add a comment that this manual patching can be removed in the next version of sublime-music (maybe with link to the upstream issue)

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 190204` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
6 packages built:
</summary>
<ul>
<li>hydrus</li>
<li>jellyfin-mpv-shim</li>
<li>plex-mpv-shim</li>
<li>python310Packages.mpv</li>
<li>python39Packages.mpv</li>
<li>sublime-music</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

<details>
hydrus:
<details>

got build log for '/nix/store/1gx5c1qvq522m1zrxys0ja0pfiwhcxdb-python3.10-hydrus-499' from 'daemon'
When evaluating attribute ‘hydrus’:
warning: duplicate-check-inputs
Dependencies listed in `propagatedBuildInputs` should not be repeated in `checkInputs`.

Detected duplicates:

- python3.10-nose-1.3.7

Near pkgs/applications/graphics/hydrus/default.nix:55:3:

```
   |
55 |   checkInputs = with python3Packages; [ nose mock httmock ];
   |   ^
```

Near pkgs/applications/graphics/hydrus/default.nix:28:3:

```
   |
28 |   propagatedBuildInputs = with python3Packages; [
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/duplicate-check-inputs.md
</details>
python310Packages.mpv:
<details>

got build log for '/nix/store/9lc9fn5kbf2dyznixbdkjmiph1k37f5z-python3.10-mpv-1.0.1' from 'daemon'
When evaluating attribute ‘python310Packages.mpv’:
warning: maintainers-missing
Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/mpv/default.nix:28:3:

```
   |
28 |   meta = with lib; {
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md
</details>
python39Packages.mpv:
<details>

got build log for '/nix/store/34bvpg4pdn97xs7zhxg41q0g591xr027-python3.9-mpv-1.0.1' from 'daemon'
When evaluating attribute ‘python39Packages.mpv’:
warning: maintainers-missing
Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/mpv/default.nix:28:3:

```
   |
28 |   meta = with lib; {
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md
</details>
</details>

File: pkgs/applications/audio/sublime-music/default.nix [hunk 1]
```diff
@@ -28,9 +28,12 @@ python3Packages.buildPythonApplication rec {
   nativeBuildInputs = [
```
Comment (onny) [inline pkgs/applications/audio/sublime-music/default.nix:28]:
```suggestion
  nativeBuildInputs = with python3Packages; [
```

File: pkgs/applications/audio/sublime-music/default.nix [hunk 2]
```diff
@@ -28,9 +28,12 @@ python3Packages.buildPythonApplication rec {
...
     gobject-introspection
     python3Packages.poetry-core
+    python3Packages.pythonRelaxDepsHook
```
Comment (onny) [inline pkgs/applications/audio/sublime-music/default.nix:31]:
```suggestion
    poetry-core
    pythonRelaxDepsHook
```

File: pkgs/applications/audio/sublime-music/default.nix [hunk 3]
```diff
@@ -28,9 +28,12 @@ python3Packages.buildPythonApplication rec {
...
     wrapGAppsHook
   ];
 
+  pythonRelaxDeps = [ "python-mpv" ];
```
Comment (onny) [inline pkgs/applications/audio/sublime-music/default.nix:35]:
```suggestion
  # Can be removed in later versions
  # https://gitlab.com/sublime-music/sublime-music/-/issues/343
  pythonRelaxDeps = [ "python-mpv" ];
```

File: pkgs/applications/audio/sublime-music/default.nix (hunk 1 above)
Comment (dali99) [reply pkgs/applications/audio/sublime-music/default.nix:28]: I don't think this is very relevant and I am not comfortable doing style changes (even if I largely agree with them) to packages I hold no stake in. As long as we don't have any published standards that should in my principled opinion be left to the "owners" of the package.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 190204` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
6 packages built:
</summary>
<ul>
<li>hydrus</li>
<li>jellyfin-mpv-shim</li>
<li>plex-mpv-shim</li>
<li>python310Packages.mpv</li>
<li>python39Packages.mpv</li>
<li>sublime-music</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

<details>
hydrus:
<details>

got build log for '/nix/store/z12r9d1fl8p7crv1ic6irgf2gx61kc0z-python3.10-hydrus-500' from 'daemon'
When evaluating attribute ‘hydrus’:
warning: duplicate-check-inputs
Dependencies listed in `propagatedBuildInputs` should not be repeated in `checkInputs`.

Detected duplicates:

- python3.10-nose-1.3.7

Near pkgs/applications/graphics/hydrus/default.nix:55:3:

```
   |
55 |   checkInputs = with python3Packages; [ nose mock httmock ];
   |   ^
```

Near pkgs/applications/graphics/hydrus/default.nix:28:3:

```
   |
28 |   propagatedBuildInputs = with python3Packages; [
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/duplicate-check-inputs.md
</details>
python310Packages.mpv:
<details>

got build log for '/nix/store/44gcl0pghvxihwx1pynzghvvzgy580f3-python3.10-mpv-1.0.1' from 'daemon'
When evaluating attribute ‘python310Packages.mpv’:
warning: maintainers-missing
Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/mpv/default.nix:28:3:

```
   |
28 |   meta = with lib; {
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md
</details>
python39Packages.mpv:
<details>

got build log for '/nix/store/1wnfjr4igw38cys90125lab3xhmfc6v3-python3.9-mpv-1.0.1' from 'daemon'
When evaluating attribute ‘python39Packages.mpv’:
warning: maintainers-missing
Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/mpv/default.nix:28:3:

```
   |
28 |   meta = with lib; {
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md
</details>
</details>

File: pkgs/applications/graphics/hydrus/default.nix [hunk 4]
```diff
@@ -10,14 +10,14 @@
...
 
   src = fetchFromGitHub {
     owner = "hydrusnetwork";
     repo = "hydrus";
     rev = "refs/tags/v${version}";
-    sha256 = "sha256-dQ6a3jys6V1ihT6q8FUaX7jOA1ZDZdX5EUy03ILk7vM=";
+    sha256 = "sha256-gsOto37++++ucpDC0ri3HhROp7v6qlHENjFvsbuyM6k=";
   };
 
   nativeBuildInputs = [
```
Comment (SuperSandro2000) [inline pkgs/applications/graphics/hydrus/default.nix:23]: Is nose really a runtime dependency?

Comment (dali99) [reply pkgs/applications/graphics/hydrus/default.nix:23]: If it ever was it isn't anymore it seems

**Comment by onny** [comment]
Thank you for your work!

**Pull request merged**